apps/hashdotai/glossary/knowledge-graphs.mdx

```diff
@@ -5,10 +5,148 @@ slug: knowledge-graphs
+
+## Property Graphs (Labeled Property Graphs)
+
+Another broad category is the *labeled property graph (LPG)* model, which is the basis of many graph databases (such as Neo4j, TigerGraph, Amazon Neptune (in LPG mode), etc.). In a property graph, nodes and edges can have **labels** and **properties** (key–value pairs) associated with them. This is a more schema-optional approach: rather than requiring a global ontology, property graphs allow each node/edge to carry its own descriptive attributes. For example, a node representing a **Person** might have properties like `name="Alice"`, and `age=”23”`, while an edge representing **purchased** might have the property `date="2025-02-28"`. Property graphs are very flexible and can represent complex networks with heterogeneous data.
+
+### Use cases and benefits
+
+Property graph knowledge graphs are common in **enterprise** settings where performance and agility are priorities. They are well-suited for operational applications like fraud detection, recommendation, or supply chain analysis where you need to traverse and query complex relationships quickly. Graph databases implementing LPG can perform graph traversals and pattern matching efficiently at scale, which may be hard to do with relational databases in such domains. Many organizations build internal knowledge graphs using property graph models to achieve a unified view of their business data (customers, products, transactions, etc.) without having to predefine a rigid global schema. For instance, a **supply chain knowledge graph** can use a property graph to model suppliers, factories, shipments, parts, and so on, with various attributes for each, enabling fast queries like “find alternative suppliers for component X in region Y” or “which products would be affected if supplier Z has a delay?”. (We will see a detailed supply chain example later.) Property graphs also lend themselves to integration with graph analytics and algorithms (community detection, shortest paths, centrality, etc.), supporting advanced **network analysis** on enterprise data. The benefit of this type is often *pragmatic flexibility*: teams can start populating a graph and iteratively add properties or new types of nodes as needed, aligning with agile development.
```

Comment:
I'm not sure it's important but I don't have a good intuition as to what 'agility' means in this context (first sentence).